language-support/scala/examples/iou-no-codegen/build.sbt

```diff
@@ -34,7 +33,7 @@ lazy val commonSettings = Seq(
     "-Xfuture",
     "-Xlint:_,-unused"
   ),
-  resolvers ++= daResolvers,
+  resolvers += Resolver.mavenLocal,
```

Comment:
So, we should delete it, because this is only something you'd do if you were building daml yourself for testing, which is not a standard workflow, to put it lightly. It's not a harmless thing to include, because it can make builds unreproducible, so unless a typical user needs it, we should delete it.